packages/devtools_app/test/inspector_v2/inspector_integration_test.dart

```diff
@@ -289,6 +290,59 @@ void main() {
     },
   );
 
+  testWidgetsWithWindowSize(
+    'tree nodes contain only essential information',
+    windowSize,
+    (WidgetTester tester) async {
+      const requiredDetailsForTreeNode = [
+        'description',
+        'shouldIndent',
+        'valueId',
+        'widgetRuntimeType',
+      ];
+      const possibleDetailsForTreeNode = [
+        'textPreview',
+        'children',
+        'createdByLocalProject',
+      ];
+      const extraneousDetailsForTreeNode = [
+        'creationLocation',
+        'type',
+        'style',
+        'hasChildren',
+        'stateful',
+      ];
+
+      await _loadInspectorUI(tester);
+      final state = tester.state(find.byType(InspectorScreenBody))
+          as InspectorScreenBodyState;
+      final rowsInTree = state.controller.inspectorTree.rowsInTree.value;
+
+      for (final row in rowsInTree) {
+        final detailKeys = row?.node.diagnostic?.json.keys ?? [];
```

Comment:
Ubernit: the default list should be `const <String>[]`

Reply:
done!